package.json

```diff
@@ -12,10 +12,11 @@
     "format": "prettier . --write"
   },
   "devDependencies": {
-    "prettier": "^2.0.0",
-    "prettier-plugin-packagejson": "^2.0.0"
+    "prettier": "^2.8.0",
+    "prettier-plugin-packagejson": "^2.3.0"
   },
   "engines": {
     "node": "^12.0.0"
```

Comment:
unrelated, but that seems a bit old.

Reply:
2.8.0 is the current version: https://www.npmjs.com/package/prettier